### ISSUE TITLE
Webhook secrets label changes is not correct in documentation

### ISSUE THREAD

**Issue by Tasteful** [external-secrets/external-secrets#3875]
The label that was added in the https://github.com/external-secrets/external-secrets/releases/tag/v0.10.0 for webhook secret is not correct or missing in the documentation.

This is places that I have found but it might be other places as well.

https://github.com/external-secrets/external-secrets/blob/adf4da46ac8351adaf48d6aa260ab80519a52923/docs/snippets/generator-webhook.yaml#L23

https://github.com/external-secrets/external-secrets/blob/adf4da46ac8351adaf48d6aa260ab80519a52923/docs/provider/webhook.md?plain=1#L32

**Comment by gusfcarvalho** [comment]
good Catch!! Would you be able to open a PR fixing it?🙏 If not I can do it this weekend I think.

**Comment by Skarlso** [comment]
If this is a problem this still needs fixing.

**Comment by Skarlso** [comment]
I believe this still needs to be addressed?

**Comment by ricardosilva86** [comment]
I am going to work on this as my first contribution to the project.

**Comment by evrardj-roche** [comment]
@ricardosilva86  what's the state of your contribution? Do you need any help? How can I help you?

**Comment by jw2340** [comment]
I can take this up as a first contribution if this still needs to be updated

**Comment by jw2340** [comment]
I think this was already fixed here https://github.com/external-secrets/external-secrets/pull/3898/changes

I don't see any more docs with the old label https://github.com/search?q=repo%3Aexternal-secrets%2Fexternal-secrets+%22generators.external-secrets.io%2Ftype%3A+webhook%22&type=code

**Comment by Skarlso** [comment]
Oh perfect! Thanks so much for investigating. :) I think, then this can be closed.
